tests/helpers/test_translation.py

```diff
@@ -192,3 +192,17 @@ async def mock_load_translation_files(files):
         "component.component1.title": "Component 1",
         "component.component1.hello": "world",
     }
+
+
+async def test_get_translation_categories(hass):
+    """Test the get translations helper loads config flow translations."""
+    with patch.object(translation, "async_get_config_flows", return_value={"light"}):
+        translations = await translation.async_get_translations(
+            hass, "en", "title", None, True
+        )
+        assert "component.light.title" in translations
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5382946)